system/lib/pthread/proxying.c

```diff
@@ -207,6 +207,12 @@ static em_task_queue* get_or_add_tasks_for_thread(em_proxying_queue* q,
   return tasks;
 }
 
+static void dummy()
+{
+}
```

Comment:
No reason to take multiple lines for this, I think.
```suggestion
static void dummy() {}
```